src/coffee/stockimport.coffee

```diff
@@ -349,15 +350,32 @@ class StockImport
     posts = _.map inventoryEntries, (entry) =>
       existingEntry = @_match(entry, existingEntries)
       if existingEntry?
-        synced = @sync.buildActions(entry, existingEntry)
-        if synced.shouldUpdate()
-          @client.inventoryEntries.byId(synced.getUpdateId()).update(synced.getUpdatePayload())
-        else
-          Promise.resolve statusCode: 304
+        @_updateInventory(entry, existingEntry)
       else
         @client.inventoryEntries.create(entry)
 
     debug 'About to send %s requests', _.size(posts)
-    Promise.all(posts)
+    Promise.map(posts, (p) -> p)
```

Comment:
quick question, can you explain the difference between this and the old code.

Reply:
I think there's no difference in this case. I was probably testing sth, but forgot to put back `Promise.all`. https://github.com/sphereio/sphere-stock-import/pull/67/commits/f64267b82292de2c9215da56b06ca083b619b9cd